fabtests/pytest/efa/test_efa_device_selection.py

```diff
@@ -46,7 +46,7 @@ def test_efa_device_selection(cmdline_args, fabric):
             cmdline_args_copy.additional_client_arguments = "-d " + client_domain_name
             cmdline_args_copy.strict_fabtests_mode = strict_fabtests_mode
 
-            test = ClientServerTest(cmdline_args_copy, command, message_size="1000", prefix_type=prefix_type, timeout=300, fabric=fabric)
+            test = ClientServerTest(cmdline_args_copy, command, message_size="1000", prefix_type=prefix_type, timeout=300, fabric="efa")
```

Comment:
it seems we should test rdm for both efa and efa-direct, and only test dgram for efa